caffe2/operators/experimental/c10/cpu/add_cpu.cc

```diff
@@ -11,14 +11,14 @@ namespace {
 
 template <class DataType>
 void add_op_cpu_impl(
-    const C10Tensor& A_,
-    const C10Tensor& B_,
-    const C10Tensor& C_,
+    const at::Tensor& A_,
+    const at::Tensor& B_,
+    const at::Tensor& C_,
     bool legacy_broadcast,
     int axis) {
-  Tensor A(A_);
-  Tensor B(B_);
-  Tensor C(C_);
+  Tensor A{C10Tensor(A_)};
+  Tensor B{C10Tensor(B_)};
+  Tensor C{C10Tensor(C_)};
```

Comment:
I've suggested we should make some free functions `to_caffe2_tensor` and `to_at_tensor` to do this conversion. This is a really good reason to do so.

Reply:
agreed. Currently, C10Tensor is the only way we have to converting tensors though, so I'll keep it here. Removing C10Tensor and adding a different way for conversion is a separate workstream.